homeassistant/components/spokestack/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""The Spokestack TTS component."""
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+PLATFORMS = ["tts"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Hello World component."""
+    # Ensure our name space for storing objects is a known type. A dict is
+    # common/preferred as it allows a separate instance of your class for each
+    # instance that has been created in the UI.
+    hass.data.setdefault(DOMAIN, {})
+
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Spokestack from a config entry."""
+    hass.data[DOMAIN][entry.entry_id] = entry.data["voice"]
+
+    hass.data[DOMAIN].update(entry.data)
+    for component in PLATFORMS:
+        hass.async_create_task(
+            hass.helpers.discovery.async_load_platform(
+                component, DOMAIN, {}, hass_config=hass.data
+            )
+        )
```

Comment:
```suggestion
    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
```

Reply:
For this one, `homeassistant.components.tts` doesn't have the attribute `async_setup_entry` so `async_setup_platforms` returns an `AttributeError`. The way I have it here works, but I'm not sure what to do. Other than adding `async_setup_entry` to the base tts component, but we've both seen how well I know this stuff :stuck_out_tongue_closed_eyes:.